components/bl-data-grid-component/component.json

```diff
@@ -59,6 +59,18 @@
       "handlerDescription": "This handler allows you to specify whether or not the floating filter is displayed.",
       "handlerLabel": "Floating Filter Logic"
     },
+    {
+      "type": "checkbox",
+      "name": "resizableColumns",
```

Comment:
resizableColumns - name for list that contains only resizable columns. 

Would be better to name this prop - resizable | isColumnsResizable